notebooks/plate_scale_plots.ipynb

```diff
@@ -0,0 +1,1184 @@
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<matplotlib.legend.Legend at 0x7fb0944be550>"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "fig = plt.figure()\n",
+    "ax1 = fig.add_subplot(211)\n",
+    "ax2 = fig.add_subplot(212)\n",
+    "plt.subplots_adjust(right=0.7,hspace=0)\n",
+    "ax1.set_xticklabels([])\n",
+    "\n",
+    "# plot the values themselves\n",
+    "for angle, radial, tangential, detector in zip(model.fieldAngles, model.radialScales,\n",
+    "                                               model.tangentialScales, detectors):\n",
+    "    line, = ax1.plot(angle, radial, label=detector)\n",
+    "    ax1.plot(angle, tangential, color=line.get_color())\n",
+    "\n",
+    "ax1.plot(model.fieldAngle, model.radialScale, '--', lw=1.5, color=\"black\", label=\"radial\")\n",
+    "ax1.fill_between(model.fieldAngle, model.radialScale-model.radialScaleStd,\n",
+    "                 model.radialScale+model.radialScaleStd,\n",
+    "                alpha=0.4, color=\"grey\")\n",
+    "\n",
+    "ax1.plot(model.fieldAngle, model.tangentialScale, '-.', lw=1.5, color=\"black\", label=\"tangential\")\n",
+    "ax1.fill_between(model.fieldAngle, model.tangentialScale-model.tangentialScaleStd,\n",
+    "                 model.tangentialScale+model.tangentialScaleStd,\n",
+    "                alpha=0.4, color=\"grey\")\n",
+    "\n",
+    "ax1.plot(cameraField, cameraRadial, '--', color='red', label='camera radial')\n",
+    "ax1.plot(cameraField, cameraTangential, '-.', color='red', label='camera tangential')\n",
+    "\n",
+    "# ax1.set_xlabel(\"field angle (deg)\")\n",
+    "ax1.set_ylabel(\"pixel size (arcsec)\")\n",
+    "ax1.set_title(\"plate scale\")\n",
+    "ax1.legend(bbox_to_anchor=(1,1.04), loc=\"upper left\")\n",
+    "\n",
+    "# differences\n",
+    "ax2.axhline(0, color=\"black\", lw=1, alpha=0.7)\n",
+    "ax2.plot(cameraField, cameraRadial-model.radialScale, ls='--', label=\"radial\")\n",
+    "ax2.plot(cameraField, cameraRadial-model.tangentialScale, ls='-.', label=\"tangential\")\n",
```

Comment:
I think this should be `cameraTangential-model.tangentialScale`

Reply:
Good catch: now the delta plot makes a lot more sense.